src/components/section/form-section.scss

```diff
@@ -3,6 +3,7 @@
 .parentResizer {
   margin-top: 0.5rem;
   margin-bottom: 1.5rem;
+  width: 90%;
```

Comment:
I have reservations about this. This class us being reused in the section as well as the obsGroup and it might create some unintended side effects. Is it possible this can be handled at the field level or only affect the obsGroup?

Reply:
@gitcliff did you see @pirupius's comment?

Reply:
thanks @pirupius i have added a styling at the field level 
<img width="727" alt="Screenshot 2024-05-05 at 11 17 11 AM" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/46714226/e99c2ffe-0fbc-4053-8ecc-70742678af24">

Reply:
@pirupius the last field is not part of the obs group type questions, its a field of obs type 
<img width="1327" alt="Screenshot 2024-05-07 at 9 56 05 AM" src="https://github.com/openmrs/openmrs-form-engine-lib/assets/46714226/6c9db187-8b48-4692-815a-b41d7a329470">

Reply:
The tooltip icon for the last field is [here](https://github.com/openmrs/openmrs-form-engine-lib/blob/6d9ba6fdebd7ee6b76767cee201876b4279b9233/src/components/section/form-section.component.tsx#L88) and for the first two fields of obs group is [here ](https://github.com/openmrs/openmrs-form-engine-lib/blob/main/src/components/group/obs-group.component.tsx#L63)